app/assets/stylesheets/availabilities.scss

```diff
@@ -83,3 +123,193 @@
   top: -2px;
   float: left;
 }
+
+.availabilities-all-cell {
+  border: 1px solid #000 !important;
+}
+
+.availability-list {
+  list-style-type: none;
+  padding-left: 0;
+  margin-bottom: 0;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+li.text-overflow-ellipsis {
+  list-style-position: inside;
+}
+
+.body {
+  text-rendering: geometricPrecision;
+}
+
+html, body {
+  height: 100%;
+}
+
+.places-container {
+  height: calc(100% - 71px);
```

Comment:
Lol, what is this 71px

Reply:
I think it's 71px is the navbar height

Reply:
@donjar I think it's an attempt at fighting CSS. Not sure @lungsin ?

Reply:
What I'm trying to say is to abstract this 71px to a separate variable or so, so that it becomes clear what the intent is

Reply:
Done